src/partition/ReceivedPartition.cpp

```diff
@@ -604,9 +647,12 @@ void ReceivedPartition::createOwnerInformation()
     }
 #endif
     auto filteredVertices = std::count(globalOwnerVec.begin(), globalOwnerVec.end(), 0);
-    if (filteredVertices)
+    if (filteredVertices) {
       PRECICE_WARN("{} of {} vertices of mesh {} have been filtered out since they have no influence on the mapping.",
                    filteredVertices, _mesh->getGlobalNumberOfVertices(), _mesh->getName());
+      if (_allowDirectAccess)
+        PRECICE_WARN("Associated data values of the filtered vertices will be filled with zero values in order to provide valid data for other participants when reading data.");
```

Comment:
You could integrate this warning in the previous message by adding a `{}` at the end.
The you can pass a empty string or this message.